### DISCUSSION TITLE
linux curses start up error: init.lua No such file

### DISCUSSION THREAD

**Discussion by clayton256** [orbitalquark/textadept#99] [General]
On Ubuntu 18.04, testadept works great. I installed both to /usr/local/bin/. However, when I Run textadept-curses I get an Initialization error "cannot open /usr/local/bin/core/init.lua: No such file or directory". 

I there something different that needs to be done touse the curses version?

Thanks!

**Comment by orbitalquark** [comment]
It looks like textadept-curses thinks that Textadept's home directory is */usr/local/bin* instead of something like */usr/local/share/textadept*. You'll have to make a symlink to */usr/local/share/textadept/textadept-curses* from */usr/local/bin*, or wherever you extracted Textadept to.
